test/syscalls/linux/write.cc

```diff
@@ -169,6 +169,9 @@ TEST_F(WriteTest, WriteIncrementOffsetAppend) {
   FileDescriptor f = ASSERT_NO_ERRNO_AND_VALUE(Open(tmpfile.path().c_str(),O_WRONLY | O_APPEND));
   int fd = f.get();
 
+  EXPECT_THAT(WriteBytes(fd, 0), SyscallSucceedsWithValue(0));
```

Comment:
What does this call do? Does it write 0 bytes? Pls write more detailed commit message, it is unclear what you want to do here.

Reply:
updated above